packages/dbml-core/types/parse/Parser.d.ts

```diff
@@ -13,6 +13,6 @@ declare class Parser {
      * Should use parse() instance method instead of this static method whenever possible
      */
     static parse(str: string, format: 'mysql' | 'postgres' | 'dbml' | 'dbmlv2' | 'schemarb' | 'mssql' | 'json'): Database;
-    parse(str: string, format: 'mysql' | 'postgres' | 'dbml' | 'dbmlv2' | 'schemarb' | 'mssql' | 'json'): Database;
+    parse(str: string | RawDatabase, format: 'mysql' | 'postgres' | 'dbml' | 'dbmlv2' | 'schemarb' | 'mssql' | 'json'): Database;
```

Comment:
Hi @Mrxyy, thanks for your interest in contributing to DBML :bow:, about this, can you use [overload function declaration](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html) in Typescript to make the signature more precise? We only allow `str` to be `RawDatabase` when `format` is `json`.

Reply:
Thank you for your suggestion.